golang.go

```diff
@@ -0,0 +1,27 @@
+// package docummentation
```

Comment:
The file has increased in 27 lines.
<!-- lookout comment separator -->
---
This line exceeded 13 chars.